catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/resource/download/ReliableResourceDownloadManager.java

```diff
@@ -181,34 +181,109 @@ public boolean isCacheEnabled() {
         return downloaderConfig.isCacheEnabled();
     }
 
+    public void setCacheEnabled(boolean cacheEnabled) {
+        downloaderConfig.setCacheEnabled(cacheEnabled);
+    }
+
     public void setProductCacheDirectory(String productCacheDirectory) {
-        this.downloaderConfig.getResourceCache().setProductCacheDirectory(productCacheDirectory);
+        this.downloaderConfig.getResourceCache()
+                .setProductCacheDirectory(productCacheDirectory);
     }
 
-    private void publishStartEvent(ResourceResponse resourceResponse,
-            Metacard metacard, String downloadIdentifier) {
-        downloaderConfig.getEventPublisher().postRetrievalStatus(resourceResponse,
-                ProductRetrievalStatus.STARTED,
-                metacard,
-                null,
-                0L,
-                downloadIdentifier);
+    public void cleanUp() {
+        executor.shutdown();
+        try {
+            executor.awaitTermination(ONE_SECOND_IN_MS, TimeUnit.MILLISECONDS);
+        } catch (InterruptedException e) {
+            executor.shutdownNow();
+        }
     }
 
-    private ResourceResponse startDownload(String downloadIdentifier, ResourceResponse resourceResponse,
-            ResourceRetriever retriever, Metacard metacard) {
+    private void publishStartEvent(Metacard metacard, ResourceResponse resourceResponse, String downloadIdentifier) {
```

Comment:
On a different subtask I removed this logic out of the manager since it didn't belong. We wanted all publish events to occur in the same place. Did things change or render that strategy moot?